plugins/modules/rds_snapshot_info.py

```diff
@@ -388,12 +378,12 @@ def main():
         ],
     )
 
-    conn = module.client("rds", retry_decorator=AWSRetry.jittered_backoff(retries=10))
+    client = module.client("rds", retry_decorator=AWSRetry.jittered_backoff())
```

Comment:
```suggestion
    client = module.client("rds")
```
are all functions not supposed to implement wrapper `@AWSRetry.jittered_backoff()`?